### PR TITLE
[WIP] Tip to see the content of a single encrypted var

### PR DESCRIPTION
**Pull request by csillab**
This tip was added by PR #37341 to the network getting started guide and it was suggested that it should be added to the main vault docs as well.

+label: docsite_pr

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

Vault documentation

##### ANSIBLE VERSION

```
$ ansible --version
ansible 2.5.0 (stable-2.5 4b406de19a) last updated 2018/03/25 13:51:08 (GMT +000)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/ubuntu/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ubuntu/ansible/lib/ansible
  executable location = /home/ubuntu/ansible/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]
```

##### ADDITIONAL INFORMATION

```
```

### CODE REVIEW

**Review by dharmabumstead — APPROVED** [review]
Doc update OK. Need a tech review from someone in ansible core.

File: docs/docsite/rst/user_guide/vault.rst [hunk 1]
```diff
@@ -193,6 +193,19 @@ Result::
 
 See also :ref:`single_encrypted_variable`
 
+After you added the encrypted value to a var file, you can see the original value by using the debug module. Please note if your YAML file defines the `ansible_connection` variable (as we used in our example), it will take effect when you execute the command below. To prevent this, please make a copy of the file without the ansible_connection variable. 
+
+.. code-block:: console
+
+   cat vars.yml | grep -v ansible_connection >> vars_no_connection.yml
```
Comment (bcoca) [inline docs/docsite/rst/user_guide/vault.rst]: useless use of cat

also .. connection does not affect debug .. so this step is really irrelevant

**Review by bcoca — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/user_guide/vault.rst (hunk 1 above)
Comment (csillab) [reply docs/docsite/rst/user_guide/vault.rst]: The reason I included the step of getting rid of connection is that it actually didn't work when it was set to network_cli...Tested in 2.5. I was surprised too.

**Comment by csillab** [comment]
I removed the reference to the ansible_connection variable.

As a side note I'd add that the ansible_connection variable does affect debug module (tested in 2.4.2.0 and 2.5.0.0).

**Comment by bcoca** [comment]
debug is an action plugin and does not connect to the target to run, same goes for things like add_host, group_by and pause. A connection object is always created, but in these cases it is never used.

**Review by gundalow — APPROVED** [review]

**Review by gundalow — DISMISSED** [review]
Needs updates as per bcoca

File: docs/docsite/rst/user_guide/vault.rst (hunk 1 above)
Comment (acozine) [reply docs/docsite/rst/user_guide/vault.rst]: @bcoca's comment was addressed in 29410428b337e067e76c7df119b5036a0bc28343

**Review by bcoca — APPROVED** [review]

**Pull request merged**